blackbox/docs/configuration.txt

```diff
@@ -60,6 +60,13 @@ For more info about table creating syntax please refer to
 
   Specifies the refresh interval of a shard in milliseconds.
 
+**write.wait_for_active_shards**
+  | *Default:* ``all``
+  | *Runtime:* ``yes``
+
+  Specifies the number of active shard copies to check for before proceeding
+  with a write operation.
```

Comment:
I think it would be great to extend this a bit. Since the default is  `all` it's unclear what the allowed values are. It's also a bit unclear if this will wait forever or if there is some kind of timeout?